### PR TITLE
Update expm1.rst

### PR DESCRIPTION
**Pull request by Lukasz-Kurylo**
Add equation clarifying operation performed.

### CODE REVIEW

File: source/elements/oneMKL/source/domains/vm/expm1.rst [hunk 1]
```diff
@@ -12,6 +12,7 @@ expm1
 
 
    Computes an exponential of vector elements decreased by 1.
+   exp(a[i]) - 1
```
Comment (rscohn2) [inline source/elements/oneMKL/source/domains/vm/expm1.rst:15]:
```suggestion
   ``exp(a[i]) - 1``
```

Format as code:
```
``exp(a[i]) - 1``
```

Format as math:
```
:math:`exp(a[i]) - 1`
```

**Comment by rscohn2** [comment]
Making a release. Please merge or close

**Comment by Lukasz-Kurylo** [comment]
Who can merge this?

**Comment by rscohn2** [comment]
@spencerpatty or @mkrainiuk can merge

**Pull request merged**